python/lsst/pipe/base/graph/graph.py

```diff
@@ -51,12 +53,16 @@
 
 # modify this constant any time the on disk representation of the save file
 # changes, and update the load helpers to behave properly for each version.
-SAVE_VERSION = 1
+SAVE_VERSION = 2
 
 # String used to describe the format for the preamble bytes in a file save
 # This marks a Big endian encoded format with an unsigned short, an unsigned
 # long long, and an unsigned long long in the byte stream
```

Comment:
This comment probably needs small update to describe versions 1 and 2 differences.